tree.txt

```diff
@@ -0,0 +1,345 @@
+[INFO] Scanning for projects...
```

Comment:
this file "tree.txt" is not required. Was this checked in by mistake?

Reply:
Removed. Was generated part of the dependency check